config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -143,4 +149,109 @@ hubs:
     template: daskhub
     auth0:
       connection: github
-    config: *stagingConfig
+    config:
+      basehub:
+        nfs:
+          enabled: true
+          pv:
+            mountOptions:
+            - soft
+            - noatime
+            # Google FileStore IP
+            serverIP: 10.229.44.234
+            # Name of Google Filestore share
+            baseShareName: /homes/
+        jupyterhub:
+          proxy:
+            https:
+              enabled: false
+          custom:
+            homepage:
+              templateVars:
+                org:
+                  name: Pangeo
+                  url: https://pangeo.io
+                  logo_url: "https://raw.githubusercontent.com/pangeo-data/pangeo/master/docs/_static/pangeo_simple_logo.svg"
+                designed_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                operated_by:
+                  name: 2i2c
+                  url: https://2i2c.org
+                funded_by:
+                  name: The Gordon and Betty Moore Foundation
+                  url: https://www.moore.org/
+          hub:
+            config:
+              Authenticator:
+                allowed_users: &prod_users
+                  - sgibson91
```

Comment:
Or rather, they should just be listed as admin users, not in both.

Reply:
Not yet. This PR is only updating the **staging** hub, and you've wandered into the config for the **prod** hub. I intended to coordinate with Ryan when we move the authentication method for prod over since it will affect his class and they'll need to be members of `pangeo-data:us-central1-b-gcp`, no? Or does that fact that Ryan has already added them as `allowed_users` not matter?

(Either, I was intending to move prod across in a second PR)

Reply:
aaah, sorry - forgot this was partially already deployed!